### ISSUE TITLE
Duplicate "TOC" file?

### ISSUE THREAD

**Issue by saprentice** [dita4publishers/dita4publishers#36]
Is there a benefit to creating a toc.xhtml when there's already a nav.xhtml? Fundamentally they are the same thing. Seems redundant. Maybe I'm missing something?

**Comment by drmacro** [comment]
It's a runtime option, but some publishers want to have both.

**Comment by drmacro** [comment]
Might be reasonable to make the default be no toc.xhtml.

**Comment by saprentice** [comment]
Yes. Default of no generated toc would be good. Default should produce the minimum required set of files. Thanks. 

...scott
